pkg/metering/v1/logs.go

```diff
@@ -0,0 +1,45 @@
+package v1
+
+import (
+	"github.com/SigNoz/signoz-otel-collector/pkg/metering"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/zap"
+)
+
+type logs struct {
+	Logger *zap.Logger
+	Sizer  metering.Sizer
+}
+
+func NewLogs(logger *zap.Logger) metering.Logs {
+	return &logs{
+		Logger: logger,
+		Sizer:  metering.NewJSONSizer(logger),
```

Comment:
this is hardcoded for `json`, in the gateway, through the encoder one can also configure `proto`, https://github.com/SigNoz/opentelemetry-gateway/blob/main/internal/processor/signozlimiterprocessor/processor.go#L91